rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/CommonRest.java

```diff
@@ -190,6 +190,28 @@ Map<String, List<String>> groupMethodsByClass(List<String> methods) {
                                                              toList())));
     }
 
+    @Override
+    public Map<String, Boolean> checkRolePermissions(String sessionId, List<String> roles) throws RestException {
+        if (roles == null || roles.isEmpty()) {
```

Comment:
you could you also CollectionUtils.isEmpty()

Reply:
ok